pkg/render/compliance.go

```diff
@@ -463,18 +462,23 @@ func (c *complianceComponent) complianceServerClusterRole() *rbacv1.ClusterRole
 				Resources: []string{"globalreporttypes", "globalreports"},
 				Verbs:     []string{"get", "list", "watch"},
 			},
-			{
-				APIGroups: []string{"authentication.k8s.io"},
-				Resources: []string{"tokenreviews"},
-				Verbs:     []string{"create"},
-			},
 			{
 				APIGroups: []string{"authorization.k8s.io"},
 				Resources: []string{"subjectaccessreviews"},
 				Verbs:     []string{"create"},
 			},
 		},
 	}
+
+	if c.installation.Spec.ClusterManagementType == operatorv1.ClusterManagementTypeManagement {
+		// For cross-cluster requests, an authentication review will be done for authenticating the compliance-server.
```

Comment:
Can you add a comment here as to why you don't need this for standalone clusters.